docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -338,49 +301,52 @@ func main() {
 </div>
 
-## Create a mock sub-part
+## Configure your robot's mock sub-part
+
+Now that you have your `fake` robotic arm, board, and motor working, add a sub-part to your robot.
 
-Now that you have your mock robotic arm working, add a mock sub-part to your robot.
+### What is a sub-part?
 
-### What is a part?
+Usually, when building a {{< glossary_tooltip term_id="robot" text="robot" >}}, you pick out a [single-board computer](/components/board/) like the [Jetson Nano](/components/board/jetson/) or [Raspberry Pi](/components/board/pi/).
+You follow the instructions in the **Setup** tab to install `viam-server` on your [board](/components/board/), and you start operating your robot with that computer, adding the [components](/components/) and [services](/services/) you want to use to that `viam-server` instance.
 
-A _{{< glossary_tooltip term_id="robot" text="robot" >}}_ in Viam consists of one or more computers combined into one logical unit.
-The bounds of a robot are usually pretty clear, but can be subjective.
-However, it's possible with Viam to create a robot that is made up of multiple computers.
-Each of these computer-controlled units is referred to as a _{{< glossary_tooltip term_id="part" text="part" >}}_.
-Most simple robots will have only one part, but they can have as many parts as needed.
+By utilizing {{< glossary_tooltip term_id="part" text="parts" >}}, you can expand upon this, chaining multiple computers together to build your robot with Viam:
 
-Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
-You can access any sub-part either directly, or through any part above it in the tree.
-Each part runs a single `viam-server` instance.
+- Each individual computer-controlled unit of a robot is called a “{{< glossary_tooltip term_id="part" text="part" >}}” in Viam.
+- Typically, simple robots have just one part, but you can have as many parts as your project requires.
+- Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
+- You can access any sub-part either directly, or through any part above it in the tree.
+- Each part runs a single `viam-server` instance.
 
 ## Configure a sub-part in the Viam app
 
-You will be creating a mock sub-part to control a motor.
-This could be anything, but imagine for the purpose of this tutorial that this motor controls a conveyor belt in front of your mock arm on an assembly line.
+Now, create a mock sub-part of another `fake` motor.
```

Comment:
But the instruction for it is in 338 - so it's not now. I think this line should just go, it's a duplicate instruction otherwise.

```suggestion
```